home/views.py

```diff
@@ -12,19 +12,33 @@ def landingpage(request):
     return render(request, 'home/landingpage.html')
 
 
-def displayQuestion(request, **kwargs):
+def displayQuestionPage(request, **kwargs):
+    form = CommentForm(request.POST)
     question = Question.objects.all().get(id=kwargs['pk'])
     sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
```

Comment:
Can use the dict's `get` method to make this simpler instead of an `if` expression

Reply:
same here, this is content of Lior's PR.

Reply:
fixed